src/libraries/System.Net.Security/src/System/Net/Security/NegoState.cs

```diff
@@ -579,10 +581,10 @@ private void StartReceiveBlob(LazyAsyncResult lazyResult)
                 message = _framer.EndReadMessage(ar);
             }
 
-            ProcessReceivedBlob(message, lazyResult);
+            ProcessReceivedBlob(message!, lazyResult);
         }
 
-        private void ProcessReceivedBlob(byte[] message, LazyAsyncResult lazyResult)
+        private void ProcessReceivedBlob(byte[] message, LazyAsyncResult? lazyResult)
```

Comment:
For this one, I think it should be fine to say `byte[]? message` here, and not need to suppress the callsites. It appears to be "normal flow" to pass null into here in order to throw an AuthenticationException.